src/ActionModal/components/ModalWithdrawRewards.vue

```diff
@@ -60,8 +60,15 @@ export default {
   computed: {
     ...mapGetters([`address`, `network`, `stakingDenom`]),
     transactionData() {
+      if (!this.claimedReward) return {}
```

Comment:
what is this for? will this break if `claimedReward` is not there?

Reply:
I just copied from the other components 🤷‍♀ 

Yes, it would break but it would in any case. I can just get rid of it

By the way, we should merge this now that luniehq/lunie-browser-extension#179 is in, since otherwise it will break sowieso with the first claim rewards transaction